src/Exception/ExceptionInterface.php

```diff
@@ -2,9 +2,11 @@
 
 namespace Laminas\EventManager\Exception;
 
+use Throwable;
+
 /**
  * Base exception interface
  */
-interface ExceptionInterface
+interface ExceptionInterface extends Throwable
```

Comment:
Discussed in other patches: while potentially a BC break, implementors are supposed to use this on exceptions (as a marker) anyway, therefore letting this through as-is.